scripts/qa_crowbarsetup.sh

```diff
@@ -2287,6 +2309,10 @@ function oncontroller_testsetup()
         complain 96 "VM not accessible in reasonable time, exiting."
     fi
 
+    if [ "$image_name" == "cirros" ]; then
+        vmip="cirros@$vmip"
```

Comment:
Would it make sense to have a variable for the ssh user, and set it at the same time as the `image_name` variable?